test/functional/tests/ioctl/common_utils.py

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright(c) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+from api.cas.cli_messages import __check_string_msg
+from core.test_run import TestRun
+
+
+interrupt_stop = [
+    r"Waiting for cache stop interrupted\. Stop will finish asynchronously\."
+]
+
+interrupt_start = [
+    r"Cache added successfully, but waiting interrupted\. Rollback"
+]
+
+load_and_force = [
+    r"cache\d+: Using \'force\' flag is forbidden for load operation\."
+]
```

Comment:
I don't think those messages should be in this file. Maybe more appropriate place would be in `api/cas/cli_messages.py`?